doc/man/8/rados.rst

```diff
@@ -199,6 +203,10 @@ To get a list object in pool foo sent to stdout::
 
        rados -p foo ls -
 
+To get a list of objects in PG 0.6::
+
+       rados ls --pgid 0.6
```

Comment:
could you move `--pgid 0.6` before `ls` ? as this option is not specific to `ls` command.